src/main/java/seedu/multitasky/storage/XmlAdaptedEntry.java

```
@@ -60,57 +61,72 @@ public XmlAdaptedEntry(ReadOnlyEntry source) {
         }
     }
 
+    /**
+     * This converts the Calendar object into a string type to be stored in XML file in a human editable format.
+     */
     public String convertDateToString(Calendar given) {
         String dateToString = df.format(given.getTime());
         return dateToString;
     }
 
+    /**
+     * This converts a String to a Calendar object to be passed back to Model.
+     */
     public Calendar convertStringToDate(String given) throws NullPointerException {
         Calendar setDate = null;
         Date toConvert = new Date();
         try {
             toConvert = df.parse(given);
+            setDate = setTheTime(toConvert);
         } catch (ParseException e) {
-            System.out.println("Invalid date!");
```

I think it's better to just declare that this method throws `ParseException` than to ignore the exception because by ignoring it a `ParseException` might be missed if it happens to be thrown one day (e.g. if a user edits the xml file wrongly).
```java
public Calendar convertStringToDate(String given) throws ParseException, NullPointerException {
```

Same goes for line 98 and 106.

---

Fixed, throwing Exceptions now